docs/azure/sdk/snippets/dependency-injection/WebApplicationBuilder/Program.cs

```diff
@@ -1,18 +1,49 @@
 ﻿#region snippet_WebApplicationBuilder
 using Azure.Identity;
+using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
 using Microsoft.Extensions.Azure;
 
 WebApplicationBuilder builder = WebApplication.CreateBuilder(args);
 
+List<string> queueNames = await GetQueueNames();
+
 builder.Services.AddAzureClients(clientBuilder =>
 {
+    // Register clients for each service
     clientBuilder.AddSecretClient(new Uri("<key_vault_url>"));
     clientBuilder.AddBlobServiceClient(new Uri("<storage_url>"));
+    clientBuilder.AddServiceBusClientWithNamespace("<your_namespace>.servicebus.windows.net");
```

Comment:
Wrap to prevent horizontal scrolling
```suggestion
    clientBuilder.AddServiceBusClientWithNamespace(
        "<your_namespace>.servicebus.windows.net");
```